src/main/java/com/labsynch/labseer/domain/StandardizationDryRunCompound.java

```diff
@@ -96,7 +89,23 @@ public class StandardizationDryRunCompound {
 
 	private String registrationComment;
 
+	@Enumerated(EnumType.STRING)
+	@NotNull
+	private SyncStatus syncStatus;
+
+	public enum SyncStatus {
+		COMPLETE,
+		NO_CHANGE_REQUIRED,
+		READY
+	}
+
+	@ManyToOne
+	@NotNull
+	@JoinColumn(name = "parent_id")
+	private Parent parent;
+	
 	public StandardizationDryRunCompound() {
+		this.setSyncStatus( SyncStatus.NO_CHANGE_REQUIRED);
```

Comment:
Defaults to not change required and we set it to READY if during dry run if it needs to be standardized (detects a compound change of some sort).